ingestion_tools/scripts/data_validation/tests/test_alignments.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, List
+
+import allure
+import numpy as np
+import pandas as pd
+import pytest
+from helper_angles import helper_angles_injection_errors
+
+
+@pytest.mark.alignment
+@pytest.mark.parametrize("dataset, run_name, aln_dir", pytest.dataset_run_alignment_combinations, scope="session")
+class TestAlignments:
```

Comment:
We should also validate the AreTomo alignments, which don't necessarily have a `tlt` file. I made [this package](https://github.com/uermel/cryoet-alignment), which we also use for ingestion, so we should also re-use it here. What I would suggest (to not block this PR) is that we consider updating the alignment validations as a separate scope of work, which I can take on, because there are a lot of technical details to be considered.

Reply:
One suggestion I would offer is to avoid using the same package for generating the alignment values in the validation process. Instead, I recommend explicitly defining it for validation (even though it might be a little redundant), as this would make it easier to identify any potential regressions.

Reply:
I agree, it's just a lot of parsing code to rewrite because the ALN files unfortunately are not simple column-based files. Totally doable though. FWIW, the parser I wrote has a bunch of tests, so maybe we are more interested in making sure those are good and trust the parser?